Calibration/HcalCalibAlgos/test/RecAnalyzerMinbias.cc

```diff
@@ -107,6 +127,9 @@ RecAnalyzerMinbias::RecAnalyzerMinbias(const edm::ParameterSet& iConfig) :
   eHighHE_              = iConfig.getParameter<double>("EHighHE");
   eLowHF_               = iConfig.getParameter<double>("ELowHF");
   eHighHF_              = iConfig.getParameter<double>("EHighHF");
+  eMin_                 = iConfig.getUntrackedParameter<double>("EMin",2.0);
+  runMin_               = iConfig.getUntrackedParameter<double>("RunMin",303441.5);
+  runMax_               = iConfig.getUntrackedParameter<double>("RunMax",304825.5);
```

Comment:
Where do these default values come from?
Does it make sense to have doubles instead of integers as run number?